Lorem.html

```diff
@@ -69,16 +81,9 @@ <h1> Lorem ipsum </h1>
         </p>
         <!--found me!-->
         <picture>
-          <source
-            type="image/svg+xml"
-            srcset="Images/sc-logo.svg"
-            media="(max width:70px)"
-          />
-          <img
-            src="images/sc-logo.png"
-            src="Images/sc-logo@1.5x.png 1.5x, Images/sc-logo@2x.png 2x"
-            alt="Sault college logo!"
-          />
+          <source type="image/svg+xml" srcset="images/sc-logo.svg">
+          <img src="Images/sc-logo.png" width="70" alt="Sault College Logo"
+              srcset="Images/sc-logo.png, Images/sc-logo@1.5x.png 1.5x, Images/sc-logo@2.png 2x">
```

Comment:
You still have quite a few merging problems from previous labs.  Please make sure you are incorporating all the changes in my pull requests.  If you would like assistance don't be shy to ask!